diff --git a/dist/index.js b/dist/index.js
index 973f16ed6bbe752f46dc417dd4be08c78526fcfb..6a6ef2412c0044ffd898110254897626857c6238 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1222,7 +1222,7 @@ var SseMCPTransport = class {
                   }
                   this.connected = true;
                   resolve();
-                } else if (event === "message") {
+                } else if ((event === void 0 || event === "message")) {
                   try {
                     const message = await parseJSONRPCMessage(data);
                     (_a4 = this.onmessage) == null ? void 0 : _a4.call(this, message);
@@ -1497,7 +1497,7 @@ var HttpMCPTransport = class {
                 const { done, value } = await reader.read();
                 if (done) return;
                 const { event, data } = value;
-                if (event === "message") {
+                if ((event === void 0 || event === "message")) {
                   try {
                     const msg = await parseJSONRPCMessage(data);
                     (_a4 = this.onmessage) == null ? void 0 : _a4.call(this, msg);
@@ -1624,7 +1624,7 @@ var HttpMCPTransport = class {
             if (id) {
               this.lastInboundEventId = id;
             }
-            if (event === "message") {
+            if ((event === void 0 || event === "message")) {
               try {
                 const msg = await parseJSONRPCMessage(data);
                 (_a4 = this.onmessage) == null ? void 0 : _a4.call(this, msg);
diff --git a/dist/index.mjs b/dist/index.mjs
index 2b80c31413f15377f1bfa76a520a914655d720d0..7298ad5e561e1847047afda222221f216144d57a 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1192,7 +1192,7 @@ var SseMCPTransport = class {
                   }
                   this.connected = true;
                   resolve();
-                } else if (event === "message") {
+                } else if ((event === void 0 || event === "message")) {
                   try {
                     const message = await parseJSONRPCMessage(data);
                     (_a4 = this.onmessage) == null ? void 0 : _a4.call(this, message);
@@ -1471,7 +1471,7 @@ var HttpMCPTransport = class {
                 const { done, value } = await reader.read();
                 if (done) return;
                 const { event, data } = value;
-                if (event === "message") {
+                if ((event === void 0 || event === "message")) {
                   try {
                     const msg = await parseJSONRPCMessage(data);
                     (_a4 = this.onmessage) == null ? void 0 : _a4.call(this, msg);
@@ -1598,7 +1598,7 @@ var HttpMCPTransport = class {
             if (id) {
               this.lastInboundEventId = id;
             }
-            if (event === "message") {
+            if ((event === void 0 || event === "message")) {
               try {
                 const msg = await parseJSONRPCMessage(data);
                 (_a4 = this.onmessage) == null ? void 0 : _a4.call(this, msg);
