diff --git a/node_modules/@weaveworksoss/backstage-plugin-flux/dist/esm/FluxEntityTable-a818f1eb.esm.js b/node_modules/@weaveworksoss/backstage-plugin-flux/dist/esm/FluxEntityTable-a818f1eb.esm.js
index b0ecc2f..f34dcd4 100644
--- a/node_modules/@weaveworksoss/backstage-plugin-flux/dist/esm/FluxEntityTable-a818f1eb.esm.js
+++ b/node_modules/@weaveworksoss/backstage-plugin-flux/dist/esm/FluxEntityTable-a818f1eb.esm.js
@@ -1181,7 +1181,7 @@ const FluxReleasesLink = ({
   const FLUX_RELEASES_URL = "https://github.com/fluxcd/flux2/releases";
   return latestFluxRelease && compare(
     (_a = latestFluxRelease == null ? void 0 : latestFluxRelease.name) == null ? void 0 : _a.substring(1),
-    resource.version.substring(1),
+    resource.version.startsWith('v') ? resource.version.substring(1) : resource.version,
     ">"
   ) ? /* @__PURE__ */ React__default.createElement(Link, { to: FLUX_RELEASES_URL }, "Update available") : null;
 };
